people.json

```diff
@@ -1,4 +1,11 @@
 [
+  {
+  "nick": "robertshepherdcpp",
+  "coordinates": [52.47867, -1.90848],
+  "links": {
+    "GitHub": "https://github.com/robertshepherdcpp/"
+    }
```

Comment:
Indentation is wrong/missing for these lines

Reply:
@GMTA Fixed now?

Reply:
No, please see the other entries to see how the indentation should look like.